wordpress-mobile/gutenberg-mobile
# Adds Clipboard Link Suggestion to Image block and Button block

Fixes #4058 

This [Gutenberg PR](https://github.com/WordPress/gutenberg/pull/35972) brings the changes from `Gutenberg` to `Gutenberg Mobile`.

To test:

Please see this [Gutenberg PR](https://github.com/WordPress/gutenberg/pull/35972) for further information.

PR submission checklist:
